tests/drivers/can/canfd/src/main.c

```diff
@@ -345,7 +345,7 @@ static void send_receive(const struct zcan_filter *filter1,
  * This test case must be run before sending/receiving test cases as it allows
  * these test cases to send/receive their own frames.
  */
-static void test_set_loopback(void)
+ZTEST(canfd, test_set_loopback)
```

Comment:
Similar to the CAN API test, this needs to happen before running the test cases. Please put it in the `setup` function and use `zassume`.